### DISCUSSION TITLE
Table row height

### DISCUSSION THREAD

**Discussion by faaa6** [quarto-dev/quarto-cli#4828] [Q&A]
Quick question.
I use quarto to render pdf documents and have very long tables. 
I would like to reduce the row height to make them a bit more dense and less spread out over multiple pages.
Is there a way to do this? Ideally as some sort of global setting.
Thank you and take care
Fabian

**Comment by mcanouil** [comment]
It is certainly possible use LaTeX since you are rendering to PDF, see <https://duckduckgo.com/?q=latex+longtable+row+height>.
Then use using partials <https://quarto.org/docs/journals/templates.html#latex-partials> or includes <https://quarto.org/docs/output-formats/pdf-basics.html#latex-includes>.
